spec/integration/driver_spec.rb

```diff
@@ -56,6 +56,54 @@ def session_url(path)
       end
     end
 
+    def capture_stdout
+      capturer = StringIO.new
+      read_io, write_io = IO.pipe
+      out_thread = Thread.new {
+        while !read_io.eof? && data = read_io.readpartial(1024)
+          capturer.write(data)
+        end
+      }
+      prev = STDOUT.dup
+      $stdout = write_io
+      STDOUT.reopen(write_io)
+      yield
+      capturer.string
+    ensure
+      STDOUT.reopen(prev)
+      $stdout = STDOUT
+      prev.close
+      out_thread.kill
+    end
+
+    it 'is threadsafe in how it captures console.log' do
+      stdout = capture_stdout do
+        pending if Capybara::Poltergeist.jruby? || Capybara::Poltergeist.rubinius?
```

Comment:
Please add a descriptive text here -  `pending "blah blah not supported" if ...`